apps/docs/src/.vitepress/components/RoadmapCard.vue

```diff
@@ -39,6 +39,9 @@ const target = computed(() => (props.href?.startsWith("http") ? "_blank" : "_sel
     border-color 0.25s,
     background-color 0.25s;
 
```

Comment:
question: is there intentionally no storybook file for this component?

Reply:
Propably not, I added one